elastic-agent-client.proto

```diff
@@ -9,6 +9,22 @@ package proto;
 option cc_enable_arenas = true;
 option go_package = "pkg/proto;proto";
 
+// Connection information sent to the application on startup so it knows how to connected back to the Elastic Agent.
```

Comment:
```suggestion
// Connection information sent to the process on startup so it knows how to connected back to the Elastic Agent.
```

Not sure if `process` is the correct one here, but we should align on one term.

Reply:
`application` is the term used through out the protocol file